XMonad/Util/Hacks.hs

```diff
@@ -32,14 +32,20 @@ module XMonad.Util.Hacks (
 
   -- * Stacking trays (trayer) above panels (xmobar)
   -- $raiseTrayer
+  trayerQuery,
   trayerAboveXmobarEventHook,
+  trayerResizeHook,
   trayAbovePanelEventHook,
+  trayResizeHook,
+  trayerResizeHook',
```

Comment:
You should put the resize functions under their own section

Reply:
i moved them down in the latest rebase.